lighthouse-core/audits/non-composited-animations.js

```diff
@@ -19,21 +19,25 @@ const UIStrings = {
   =1 {# animated element found}
   other {# animated elements found}
   }`,
-  /** Name of a compositor failure reason where the CSS property being animated is not supported on the compositor. */
-  unsupportedCSSProperty: 'Unsupported CSS Property',
+  /**
+   * @description Name of a compositor failure reason where the CSS property being animated is not supported on the compositor.
+   * @example {height, width} properties
+   */
+  unsupportedCSSProperty: 'Unsupported CSS Properties: {properties}',
```

Comment:
Can this be pluralized like `displayName`?

Reply:
ayup we can use this pattern https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/font-display.js#L27-L37